### PR TITLE
Fixing bugs for news page and filters

### PR DESCRIPTION
**Pull request by KimAT25**
List of fixed bugs:

- [Display news, List view] Title height (1 row) is 28 px instead of 32 px. #816

- [Display news, Gallery view] Size of tag 'News' and distance between Content and Date are not correct according to mockup. #773

- [Eco news] Filters can be open as a link #695

- not wraping text and title in gallery and list view, when text and title where inputted without spaces

### CODE REVIEW

File: src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.ts [hunk 1]
```diff
@@ -29,14 +29,8 @@ export class NewsListListViewComponent implements OnInit, AfterViewChecked {
 
   constructor(private renderer: Renderer2) { }
 
-  ngOnInit() {
-    this.textValidationOfMinCharacters();
-  }
+  ngOnInit() {}
```
Comment (mplesha) [inline src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.ts:32]: Do you use this lifecycle hook? What is the stile to write empty methods? (and there's **import** and **implements** )

**Comment by mplesha** [comment]
In the component <code>news-list-list-view.component.ts</code> must definitely be rewritten this part:
```js
if (titleHeight < linesQuantity.smSize) {
      elemHeight = linesQuantity.lSize;
    } else if (titleHeight > linesQuantity.lSize) {
      elemHeight = linesQuantity.hiddenSize;
    } else if (titleHeight > linesQuantity.smSize &&
      titleHeight <= linesQuantity.mSize) {
      elemHeight = linesQuantity.msSize;
    } else if (titleHeight > linesQuantity.mSize &&
      titleHeight <= linesQuantity.lSize) {
      elemHeight = linesQuantity.sSize;
    }
```
It's so complicated and convoluted. Look at here, e.g. you write:
```js
...
if (titleHeight > linesQuantity.lSize) {
...
} else if (titleHeight > linesQuantity.mSize &&
      titleHeight <= linesQuantity.lSize)
...
```
What? <code>titleHeight <= linesQuantity.lSize</code>? What other it will be if it's not <code>titleHeight > linesQuantity.lSize</code>?
Other question - what about <code>titleHeight === linesQuantity.smSize</code> (you have cases '<' and '>'  but not =)

**Comment by mplesha** [comment]
Other question: why did you put <code> ngAfterViewChecked() </code> at the bottom? See here [Style 05-14](https://angular.io/guide/styleguide) about member sequence.

File: src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.ts (hunk 1 above)
Comment (KimAT25) [reply src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.ts]: Fixed

**Comment by KimAT25** [comment]
Fixed all comments and making refactoring of if-else-if

**Pull request merged**